.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "Api"]
-	path = Api
-	url = https://github.com/aneoconsulting/ArmoniK.Api.git
```

Comment:
the submodule is only required for the StreamWrapper?

Reply:
No, but I made packages in ArmoniK.Api. Therefore, I used them instead of the submodule

Reply:
So if we remove it as submodule in Core we need to document the requirement (in case someone clones the repo)

Reply:
The package is included in the .csproj so this makes it a requirement already.

Reply:
Ah ok!